routes/api/users.js

```diff
@@ -47,4 +49,181 @@ const Users = require('../../controllers/Users/userController');
  */
 router.get('/', Users.listUsers);
 
+/**
+ * @swagger
+ * /profiles/:id/follow:
```

Comment:
`/profiles` 
Why not remove the `profiles` part on these routes? Doing this makes the routes more straight forward and it makes more logical meaning.

Reply:
Fixed